src/megaclient.cpp

```diff
@@ -4177,6 +4177,15 @@ bool MegaClient::procsc()
                                 // last acknowledged
                                 sc_la();
                                 break;
+
+                            case MAKENAMEID2('u', 'b'):
+                                // business account update
+                                // `ub` AP has no payload, so we need to set pos
+                                // one character back to avoid to consume `}` character
+                                // and close object properly
+                                sc_ub();
+                                jsonsc.pos--;
```

Comment:
It looks like we would have called `getnameid()` on 'ub' which doesn't consume the '}', how are we iterating past that one?  It's pretty unusual to adjust the `pos` directly, I think it shouldn't be necessary? thanks